src/parallel/parallel.cpp

```diff
@@ -151,6 +156,24 @@ ParallelRobinHoodHashTable::find(KeyType key)
     // TODO
 }
 
+void
+add_thread_lock_manager()
```

Comment:
```suggestion
ParallelRobinHoodHashTable::add_thread_lock_manager()
```

Member function missing parent class scope.

Reply:
My bad, good catch